packages/fetch/src/index.ts

```diff
@@ -94,7 +94,7 @@ ${
     ? `${stringify(override?.requestOptions)?.slice(1, -1)?.trim()}`
     : '';
   const fetchMethodOption = `method: '${verb.toUpperCase()}'`;
-  const fetchHeadersOption = body.contentType
+  const fetchHeadersOption = body.contentType && !isFormData
```

Comment:
does this need to all case of ` multipart/form-data`? 
`isFormData` is only valid when you set the `formData` option in config.

Reply:
Ah I see. I think I need to change the code to detect if the formData is within the API spec instead of using this boolean then. Didn't realize the boolean is for the override config

Reply:
@jamesleeht 

So, what do you think about this update?

```suggestion
  const ignoreContentTypes = ['multipart/form-data']
  const fetchHeadersOption = body.contentType && ignoreContentTypes.includes(body.contentType)
```

This update excludes content-type in all cases using `multipart/form-data`.

Reply:
LGTM!